internal/util/validation/validation.go

```diff
@@ -107,6 +113,68 @@ func ValidateBearerToken(token *string, path string) []error {
 	return asErrors(errs)
 }
 
+func ValidateCSRUsages(u *[]string) []error {
+	errs := field.ErrorList{}
+	requiredAllOf := []string{"client auth", "CA:false"}
+	notAllowed := []string{}
```

Comment:
The notAllowed are currently empty.  Why is it needed?

Reply:
Mainly for the future so that we recognize it is an option to add some values that aren't allowed.